### DISCUSSION TITLE
Multiple Subdomains

### DISCUSSION THREAD

**Discussion by curvingedge** [willswire/unifi-ddns#27] [Q&A] [answered]
Would this allow updating multiple subdomains using comma separation in the controller?

**Comment by nmbrg** [comment]
id like to know this as well. couldnt find any documentation about if it was possible or how to do it if it is.
since its not possible to add multiple DDNS settings with the same provider (dyndns) in unifi and i only want to update subdomains.

**Reply by 51av0sh** [reply]
In case you guys didn't know his by now. This feature was added last year and i works fine. 

https://github.com/workerforce/unifi-ddns/pull/28

**Comment by willswire** [comment, marked as answer]
If you're looking to update multiple subdomains on the same zone (second-level domain), then I would advise adding CNAME entries to point secondary subdomains the primary subdomain (or domain) you're targeting with DDNS.

For example, if I want to update `ftp.willswire.com` and `home.willswire.com`, I would just configure DDNS for `home.willswire.com` and add a CNAME entry for `ftp.willswire.com` to point to `home.willswire.com`.

The latest version of this project (v3.0.x) no longer includes support for updating multiple subdomains.

If you're looking to configure multiple zones for different WAN connections, check out the FAQ in `/docs/faq.md` within the project.

**Reply by curvingedge** [reply]
Thanks. This is exactly how I ended up setting it up. All the subs point to the ddns domain with CNAME.
